example_template.md

```diff
@@ -0,0 +1,102 @@
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+- [Run the EXAMPLE NAME](#run-the-example-name)
+  - [🗝️ Expected knowledge](#Expected-knowledges)
+  - [🔮 Overview of the files](#Overview-of-the-files)
+  - [🏃 Run the Flows](#run-the-flows)
+  - [🌀 Flow Diagram](#flow-diagram)
+  - [🌟 Results](#results)
+  - [🧞 OPTIONAL: EXTRA KNOWLEDGE FROM THIS EXAMPLE](#optional)
+  - [💫 Deploy with Docker](#Deploy-with-docker)
+  - [Next steps](#next-steps)
+  - [Community](#community)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Run the EXAMPLE NAME
+
+Write a short description of
+1. What is this?
+2. What is the expected outcome?
+3. What the user will learn?
+
+If you want to run the example with Docker, check the instruction at the end of this README.  
+
+## 🗝️ Expected knowledge
+
+Write a list of whatever the user is expected to know.
+
+1. Read up on [Jina 101](http://101.jina.ai).
+2. Python knowledge.
+
+### Install requirements
```

Comment:
Add a section before this for:
1. cloning the repo and cd into directory
2. Dataset download

Reply:
Second this